### PR TITLE
Add create linode page skeleton

### PR DESCRIPTION
**Pull request by SirCmpwn**
Also rips out most of the old code for this flow, and some other unused components. Also makes the card+tabs styles from the details page generic.

Closes #175.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
SirCmpwn sets the record for most test coverage added by one pull request!

**Comment by eatonphil** [comment]
Holy test coverage, Batman!

**Comment by eatonphil** [comment]
Cool. Looks good.

**Comment by eatonphil** [comment]
Ready for merge?

**Comment by SirCmpwn** [comment]
Yep.

File: scss/components/card.scss [hunk 1]
```diff
@@ -5,15 +5,67 @@
...
         bottom: 15px;
         right: 15px;
     }
 
+    .card-body {
+        padding: 2rem;
+    }
+
     header {
+        $padding: $main-padding-tb * 1.5;
```
Comment (eatonphil) [inline scss/components/card.scss]: I do not agree with this. But I will let it go right now.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**